com.unity.cinemachine/Runtime/Behaviours/CinemachineStateDrivenCamera.cs

```diff
@@ -257,7 +257,7 @@ struct HashPair { public int parentHash; public int hash; }
         int LookupFakeHash(int parentHash, AnimationClip clip)
         {
             if (m_HashCache == null)
-                m_HashCache = new();
+                m_HashCache = new Dictionary<AnimationClip, List<HashPair>>();
```

Comment:
Why did you change this?

Reply:
I felt it's more clear/readable this way